src/Tribe/Data_API.php

```diff
@@ -387,10 +391,16 @@ protected function get_attendees( $post_id, $context, $services = false ) {
 		if ( ! isset( $services['class'] ) && is_numeric( $post_id ) ) {
 			return Tribe__Tickets__Tickets::get_event_attendees( $post_id );
 		} elseif ( ! isset( $services['class'] ) && ! is_numeric( $post_id ) ) {
-			return array();
+			return [];
+		}
+
+		if( ! empty( $services['class'] ) ) {
+			$provider = Tribe__Tickets__Tickets::get_ticket_provider_instance( $services['class'] );
 		}
 
-		$provider = call_user_func( array( $services['class'], 'get_instance' ) );
+		if( empty( $provider ) ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('

<sub>hash: 322f922645fa53f8a8498c936feba7b2</sub>